opensrp-giz-malawi/build.gradle

```diff
@@ -103,6 +103,19 @@ android {
         targetCompatibility 1.8
     }
 
+    signingConfigs {
+        release {
+
+            v1SigningEnabled false
+            v2SigningEnabled true
+
+            keyAlias System.getenv("KEYSTORE_ALIAS")
+            keyPassword System.getenv("KEY_PASSWORD")
+            storePassword System.getenv("KEYSTORE_PASSWORD")
+            storeFile file(System.getProperty("user.home") + "/<project>.keystore.jks")
```

Comment:
Needs to be the actual project key store file here

Reply:
Also should tally with the value in your CI file in the **Decode Keystore file** step